### ISSUE TITLE
Slick slider doesn't meet Accessibility Standards! Is there any solution on that?

### ISSUE THREAD

**Issue by BrunildaDosti** [Accessible360/accessible-slick#83]
Hi,

I'm using the Slick Slider module into one of my sites which should meets accessibility standards wherever possible.
I'm using WAVE tool and I see that many errors come from Slick Slider. 
Is there any guide or detailed instructions on how to fix those errors coming from this module and ensure that it meets the accessibility standards, please?

Thanks in advance,
Brunilda
<img width="658" alt="Screenshot 2023-02-07 at 4 37 20 PM" src="https://user-images.githubusercontent.com/23187783/217290853-97e2ebb6-b362-4934-be30-d27cab774aec.png">
